pydis_site/apps/api/models/bot/tag.py

```diff
@@ -67,7 +67,7 @@ def validate_tag_embed_footer(footer):
             validator(value)
 
 
-def validate_tag_embed_author(author):
+def validate_tag_embed_author(author: Any) -> None:
```

Comment:
As per [Discord docs](https://discordapp.com/developers/docs/resources/channel#embed-object-embed-author-structure) & like earlier comments:

```suggestion
def validate_tag_embed_author(author: Dict[str, str]) -> None:
```